Trace 0:
  Transitions:
    'calling PyErr_Clear()'
    'returning'
  Return value:
    repr(): PointerToRegion(gcctype='struct PyObject *', loc=gcc.Location(file='tests/cpychecker/refcounts/PyErr_Clear/correct/input.c', line=30), region=RegionForGlobal(gcc.VarDecl('_Py_NoneStruct')))
    str(): (struct PyObject *)&RegionForGlobal(gcc.VarDecl('_Py_NoneStruct')) from tests/cpychecker/refcounts/PyErr_Clear/correct/input.c:30
    r->ob_refcnt: refs: 1 owned, 1 borrowed
    r->ob_type: None
  Exception:
    (struct PyObject *)0 from tests/cpychecker/refcounts/PyErr_Clear/correct/input.c:29
